app/lang/en.json

```diff
@@ -35,22 +35,27 @@
     "firstNames": "First names",
     "lastName": "Last name",
     "otherNames": "Any other names you're known by (optional)",
+    "otherNamesLabel": "Other names",
     "dateOfBirth": "Date of birth",
     "dateOfBirthHint": "For example, 31 3 1980",
-    "day": "Day",
-    "month": "Month",
-    "year": "Year",
-    "enterFirstNames": "Enter first names",
-    "firstNamesTooLong": "Enter a first name that's less than 54 characters long",
-    "enterLastName": "Enter last name",
-    "lastNameTooLong": "Enter a last name that's less than 62 characters long",
-    "otherNamesTooLong": "Enter other names that are less than 51 characters long",
-    "enterDateOfBirth": "Enter date of birth",
-    "dateOfBirthMustBeReal": "Date of birth must be a real date",
-    "dateOfBirthIsFuture": "Date of birth must be before today",
+    "day": "day",
+    "month": "month",
+    "year": "year",
```

Comment:
I get why these have been lowercased but when we rely on them for form labels we're getting:

<img width="302" alt="Screenshot 2023-02-01 at 13 26 44" src="https://user-images.githubusercontent.com/17926619/216056621-739f23dc-e057-4725-a161-9a2e6ac49562.png">

I guess we just need to capitalise in `date.gohtml`?

Reply:
good spot